features/steps/steps.py

```diff
@@ -0,0 +1,43 @@
+import time
+
+from behave import given, then, when
+from django.contrib.auth.models import User
+from seleniumlogin import force_login
+
+from bdd.models import Item
+
+
+@given(u'the following items')
+def set_up_items(context):
+    Item.objects.bulk_create([
+        Item(pk=row['pk'], sort_order=i)
```

Comment:
This needs to be `i+1`, to avoid populating the item with a `0` (which is not a valid value).